bin/crew

```diff
@@ -731,10 +731,22 @@ def download
 
     when /^SKIP$/i
       Dir.mkdir @extract_dir
-
-    # Source URLs which end with .git are git sources.
-    when /\.git$/i
+    when /\.git$/i # Source URLs which end with .git are git sources.
       @git = true
+    else
+      Dir.mkdir @extract_dir
+      # use curl if CURL environment variable set
```

Comment:
Shouldn't this say `use downloader if CURL environment variable set`?

Reply:
If the `CURL` variable is set by the user, it might contain some `curl` specific environment variables and `curl` will be called instead.

Reply:
Renamed it to `CREW_USE_CURL`